content/docs/develop/operate/k8s_support/index.md

```diff
@@ -20,18 +20,25 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
 <!-- use https://www.tablesgenerator.com/markdown_tables# for editing -->
 
 | Keptn Version /<br>Installation       | Kubernetes      | AKS             | EKS         | GKE             | OpenShift | Minikube              | MicroK8s<br>(experimental) | Minishift<br>(experimental) |
-|---------------------------------------|-----------------|-----------------|-------------|-----------------|-----------|-----------------------|----------------------------|-----------------------------|
-| 0.6.x / <br>Control & Execution Plane | 1.15.x - 1.13.x | 1.15.x (1)   | 1.15 - 1.14 | 1.15.x - 1.14.x | 3.11      | -                     | -                          | 1.34.2<br>(K8s: 1.11.x)     |
-| 0.6.x / <br>Control Plane             | 1.16.x - 1.13.x | 1.16.x - 1.15.x | 1.15 - 1.14 | 1.15.x - 1.14.x | 3.11      | 1.2 <br>(K8s:1.15.x) | 1.18.x                     | 1.34.2<br>(K8s: 1.11.x)     |
+|---------------------------------------|:-----------------|:-----------------|:-------------|:-----------------|:-----------|:-----------------------|:----------------------------|:-----------------------------|
+| **0.7.x** / <br>Control & Execution Plane | 1.18.x<br>1.17.x<br>1.16.x<br>1.15.x<br>1.14.x | 1.16.x<br>1.15.x<br>see: (1) | 1.16<br>1.15<br>1.14 | 1.15.x<br>1.14.x<br>see: (3) | 3.11      | -                     | -                          | 1.34.2 (K8s: 1.11.x)     |
+| **0.7.x** / <br>Control Plane             | 1.18.x<br>1.17.x<br>1.16.x<br>1.15.x<br>1.14.x | 1.16.x<br>1.15.x<br>see: (1) | 1.16<br>1.15<br>1.14 | 1.16.x<br>1.15.x<br>1.14.x | 3.11      | 1.10.1 (K8s:1.18.2)<br>1.9.2 (K8s:1.18)<br>1.7.3 (K8s:1.17.3)<br> 1.6.2 (K8s:1.17)<br> 1.4 (K8s:1.16)<br>1.3.1 (K8s:1.15.x) | 1.18.x                     | 1.34.2 (K8s: 1.11.x)     |
+| 0.6.x / <br>Control & Execution Plane | 1.15.x<br>1.14.x<br>1.13.x | 1.15.x<br>see: (1)   | 1.15<br>1.14<br>see: (2) | 1.15.x<br>1.14.x<br>see: (2) | 3.11      | -                     | -                          | 1.34.2 (K8s: 1.11.x)     |
+| 0.6.x / <br>Control Plane             | 1.16.x<br>1.15.x<br>1.14.x<br>1.13.x | 1.16.x<br>1.15.x<br>see: (1)  | 1.15<br>1.14<br>see: (2) | 1.15.x<br>1.14.x<br>see: (2) | 3.11      | 1.2 (K8s:1.15.x) | 1.18.x                     | 1.34.2 (K8s: 1.11.x)     |
 
-**Remarks:**
+**Remarks for Keptn 0.7**
 
 * (1): AKS with K8s version before 1.15 might fail [#1429](https://github.com/keptn/keptn/issues/1429), due to a known AKS issue: [#69262](https://github.com/kubernetes/kubernetes/issues/69262)
+* (3): GKE does not provide K8s 1.16 clusters (checked on: 13th May, 2020)
+
+**Remarks for Keptn 0.6**
+
+* (1): AKS with K8s version before 1.15 might fail [#1429](https://github.com/keptn/keptn/issues/1429), due to a known AKS issue: [#69262](https://github.com/kubernetes/kubernetes/issues/69262)
+* (2): EKS and GKE do not provide K8s 1.16 clusters (checked on: 9th April, 2020)
```

Comment:
not sure if we should do the split down here, I think it's enough to have the remarks (1), (2) and (3) (in that order), without the title of the version

Reply:
Removed remark (3) since GKE 1.16 is available. Thus, this version needs to be supported by Keptn 0.7.0. 

This also makes the split of the remarks obsolete.